buildspec.yml

```diff
@@ -6,29 +6,34 @@ phases:
       golang: 1.13
   pre_build:
     commands:
-      # GOPATH is setup like the following in Codebuild standard 2.0 image
+      # GOPATH is setup like the following in Codebuild standard 4.0 image
       # /go:/codebuild/output/src<some numbers>
-      # so we copy all the source code to the appropriate location before
-      # invoking any go command
+      # so we copy all the source code to the go root install location before
+      # invoking any go command. 
+      # this also allows us to build using dep while the transition to go modules is ongoing
       - ls -lah
       - mkdir -p /go/src/github.com/aws/amazon-ecs-cli
       - cp -R . /go/src/github.com/aws/amazon-ecs-cli/
-      - |
-        env -i PATH=$PATH GOPATH=`go env GOPATH` GOROOT=`go env GOROOT` GOCACHE=`go env GOCACHE` \
-        go test -race -v -cover github.com/aws/amazon-ecs-cli/ecs-cli/modules/...
+      - cd /go/src/github.com/aws/amazon-ecs-cli
```

Comment:
In general, I avoid `cd` into any specific directory because we might forget to cd out before executing later commands. But I don't have a strong preference on this.